lib/model/CIndividualModel.cc

```diff
@@ -378,13 +382,35 @@ bool CIndividualModel::doAcceptRestoreTraverser(core::CStateRestoreTraverser& tr
                         std::cref(this->params()), std::placeholders::_1)))
         RESTORE(MEMORY_ESTIMATOR_TAG,
                 core::CPersistUtils::restore(MEMORY_ESTIMATOR_TAG, m_MemoryEstimator, traverser))
+        RESTORE_BUILT_IN(UPGRADING_PRE_7_5_STATE, upgradingPre7p5State)
     } while (traverser.next());
 
+    const double DEFAULT_CUTOFF_TO_MODEL_EMPTY_BUCKETS{0.2};
```

Comment:
For the benefit of anyone looking at this in the future, this is a local reinstatement of https://github.com/elastic/ml-cpp/pull/721/files#diff-d253406bdc862c2c89a29e7560f828a8L70